src/lib/isCI.ts

```diff
@@ -42,3 +33,12 @@ export default function isCI() {
   /* istanbul ignore next */
   return (ci.isCI && !isTest()) || !!process.env.TEST_RDME_CI;
 }
+
+/**
+ * Small env check to determine if we're in a GitHub Actions environment.
+ *
+ * @see {@link https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables}
+ */
+export function isGHA() {
+  return isCI() && ci.GITHUB_ACTIONS;
```

Comment:
Had to add this `isCI() &&` clause to avoid false positives in a GitHub Actions test runner environment unless we're explicitly testing it code that's specific to GHA. A few tests in `__tests__/lib/fetch.test.ts` started breaking when upgrading to `jest@29.4`... honestly I'm a bit confused now why they were passing in the first place 😵‍💫